service_configuration_lib/spark_config.py

```diff
@@ -45,8 +45,10 @@
 DEFAULT_DRA_MIN_EXECUTOR_RATIO = 0.25
 
 EXECUTOR_MEMORY_WARN_GB = 28
-EXECUTOR_MEMORY_CAP_GB = 120
 EXECUTOR_CORES_CAP = 12
+EXECUTOR_MEMORY_CAP_GB = 110
```

Comment:
Let's rename these variables so that they are easier to understand.
Maybe something like:
EXECUTOR_MEMORY_CAP_GB or MAX_EXECUTOR_MEMORY_ALLOWED
EXECUTOR_MEMORY_UPBOUND_GB --> MEDIUM_EXECUTOR_MEMORY
RECOMMENDED_EXECUTOR_MEMORY = 28g
Maybe we can keep: DEFAULT_EXECUTOR_MEMORY same as RECOMMENDED_EXECUTOR_MEMORY so that all jobs who don't specify executor memory will have standard 28g memory and 4 cores. 
Thoughts?

Reply:
Sounds good! I am also changing `EXECUTOR_MEMORY_CAP_GB` to `MAX_EXECUTOR_MEMORY`, and constants for cpu as well to align with other names.

Reply:
For default executor memory, will it cost more if we change it from 4g to 28g? Not sure how many jobs are using the default value.